lib/chef/provider/user/smartos.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Author:: Dave Eddy (<dave@daveeddy.com>)
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'chef/provider/user/useradd'
+
```

Comment:
Will add, thanks.

Is there a place I can read up on what the difference is? Is adding this `require` statement the only thing that is required, or do my calls to `shell_out!` need to be modified as well?